### PR TITLE
Integration proofreading

### PR DESCRIPTION
**Pull request by quinntracy**

### CODE REVIEW

File: content/en/docs/refguide/modeling/integration/odata-services/consumed-odata-services/consumed-odata-service-requirements.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ The most commonly used attribute types can be used in your app. The types of the
 Support for consuming attributes of complex types was introduced in Mendix version 10.6.
 {{% /alert %}}
 
-Complex types are not supported by the Mendix domain model. Mendix does, however, allow you to read external entities that contain attributes of a complex type by importing the properties of the complex type as attributes of the containing entity.
+Complex types are not supported by the Mendix domain model. However, Mendix allows you to read external entities that contain attributes of a complex type by importing the properties of the complex type as attributes of the containing entity.
```
Comment (quinntracy) [inline content/en/docs/refguide/modeling/integration/odata-services/consumed-odata-services/consumed-odata-service-requirements.md]: Can this small paragraph be updated/removed due to the alert box above?

Comment (quinntracy) [reply content/en/docs/refguide/modeling/integration/odata-services/consumed-odata-services/consumed-odata-service-requirements.md]: @joostverhoog can you advise?

Comment (joostverhoog) [reply content/en/docs/refguide/modeling/integration/odata-services/consumed-odata-services/consumed-odata-service-requirements.md]: No, the alert box indicates that this paragraph contains information for Studio Pro 10.6 and later. The paragraph describes that OData complex type properties become 'normal' properties in the domain model.

Comment (joostverhoog) [reply content/en/docs/refguide/modeling/integration/odata-services/consumed-odata-services/consumed-odata-service-requirements.md]: I don't like how this text uses Mendix (the company) where Studio Pro (the product) would be more precise.

Comment (quinntracy) [reply content/en/docs/refguide/modeling/integration/odata-services/consumed-odata-services/consumed-odata-service-requirements.md]: Got it, I'll keep the alert box as is and adjust from Mendix to SP. Thanks!

**Pull request merged**